### DISCUSSION TITLE
Mail "from name" in config instead of fixed 'Rallly'

### DISCUSSION THREAD

**Discussion by bmaster001** [lukevella/rallly#992] [Ideas]
Outgoing mails are always from the fixed name 'Rallly', while the email address is of course customizable in the configfile. Is there a reason that the name in not a configurable setting?

**Comment by lukevella** [comment]
It's not documented but you can modify the from name by setting `NOREPLY_EMAIL` to `Sender Name sender@server.com`. I'll create an issue to include this information in the configuration options.

**Comment by bmaster001** [comment]
Ugh, looks like I'm doing something wrong because the name doesn't change in the mails I receive.
I've tried things like
`NOREPLY_EMAIL="sender name email@server.com"`
or
`NOREPLY_EMAIL="sender name <email@server.com>"`
Maybe there's another syntax?

**Comment by lukevella** [comment]
Looks like I misspoke. The sender name [cannot be configured at the moment](https://github.com/lukevella/rallly/blob/main/apps/web/src/utils/emails.ts#L13-L20).

It's a sensible request to be able to change it. That said, this is very low priority so can't say when I will have time to work on this. I'll adjust the issue accordingly.

**Reply by jirutka** [reply]
It can be now, thanks to #1023, so this discussion can be closed.
